src/main/java/card/issuer/CardIssuers.java

```diff
@@ -0,0 +1,39 @@
+package main.java.card.issuer;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+public class CardIssuers {
+    private static Map<String, CardIssuerParameters> cardIssuersMap =  new HashMap<String, CardIssuerParameters>();
```

Comment:
Diamond operator is enough 
```suggestion
    private static Map<String, CardIssuerParameters> cardIssuersMap =  new HashMap<>();
```
In this case, the issuer name was duplicated (as a map key and name in CardIssuerParameters)